src/components/footer/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+//todo здесь нужно сделать импорт socialMediaIcons из файла '../../constants'
+import {socialMediaIcons} from '../../constants/index';
+//todo здесь нужно сделать импорт стилей из './Footer.scss'
+import './Footer.scss';
+import {author} from "../../constants/index";
+
+const FB =  socialMediaIcons[0];
+const Insta =  socialMediaIcons[1];
```

Comment:
Also, such constants should be named in camel case https://ru.wikipedia.org/wiki/CamelCase